src/pages/ImportWorkflow/ImportWorkflow.js

```diff
@@ -85,9 +85,10 @@ export const ImportWorkflow = ({ path, version, source }) => {
         method_output_mappings: [],
       };
       const res = await Ajax(signal).Cbas.methods.post(appUrls.cbasUrl, postRequestBody);
+      const methodId = res.method_id;
 
       await setAzureCookieOnUrl(signal, appUrls.cbasUiUrl, true);
-      window.location = `${appUrls.cbasUiUrl}#submission-config/${res.method_id}`;
+      Nav.goToPath('workspace-workflows-app-submission-config', { name, namespace, methodId });
     } else {
       throw new Error(
         'Error identifying a unique and valid Cromwell App. Cromwell Apps are valid if they were created by the current user in the workspace and are in a Running state.'
```

Comment:
Minor, are we rebranding this as "Workflows App"

Reply:
So: `workflows-app-submission-config` ?